VideoLocker/res/values/dimens.xml

```diff
@@ -37,10 +37,20 @@
     <dimen name="default_row_text">12sp</dimen>
     <dimen name="course_card_height">190dp</dimen>
     <dimen name="course_detail_card_height">70dp</dimen>
-    <dimen name="dashboard_navigation_element_height">85dp</dimen>
     <dimen name="large_indicator_icon_width">100dp</dimen>
     <dimen name="large_indicator_icon_height">100dp</dimen>
 
+    <dimen name="dashboard_nav_element_height">85dp</dimen>
+    <dimen name="dashboard_nav_element_left_margin">30dp</dimen>
+    <dimen name="dashboard_nav_element_right_margin">30dp</dimen>
+    <!--NOTE: This margin needs to be 75dp as per design doc
+        But, since we are using a LinearLayout in the base layout {@link layout/row_course_dashboard_list}
+        so we deduct the element's left margin i.e. {@link dashboard_nav_element_left_margin} and icon
+        width i.e. {@link #fa_xx_large} from 75dp.
+        in short: 75 - 30 - 24 = 21dp
+    -->
+    <dimen name="dashboard_nav_element_text_left_margin">21dp</dimen>
```

Comment:
It seems unlikely that the design document is referring to vertical margins given that the icon is vertically centered in the row. The iOS implementation also seems to be based on the logic that is implemented here. Maybe this can be clarified from the UI designer though. cc @bguertin, @aleffert